web3/tracing.py

```diff
@@ -0,0 +1,123 @@
+from typing import (
+    Callable,
+    List,
+    Optional,
+    Tuple,
+    Union,
+)
+
+from eth_typing import (
+    HexStr,
+)
+from eth_utils import (
+    is_checksum_address,
+)
+from eth_utils.toolz import (
+    assoc,
+)
+
+from web3._utils.rpc_abi import (
+    RPC,
+)
+from web3.method import (
+    Method,
+    default_root_munger,
+)
+from web3.module import (
+    Module,
+)
+from web3.types import (
+    BlockIdentifier,
+    BlockTrace,
+    FilterParams,
+    FilterTrace,
+    TraceMode,
+    TxParams,
+    _Hash32,
+)
+
+
+class Tracing(Module):
+    """
+    https://openethereum.github.io/JSONRPC-trace-module
+    """
+
+    _default_block: BlockIdentifier = "latest"
+
+    """ property default_block """
```

Comment:
We can prob remove this when we bring the code back in

Reply:
I see it's still used in `trace_call_munger` so I didn't want to remove it

Reply:
Oh sorry this was a tiny nit about the docstring since it doesn't actually say very much (not the property!)... we can keep it or leave it, it doesn't matter. Just might be nice to get rid of.

Reply:
Understood, done! 👍🏻